src/main/kotlin/ShoppingSystem.kt

```diff
@@ -0,0 +1,77 @@
+package shoppingsystem
+
+class Product(var name: String, var price: Double, var quantityPurchased: Int) {
+
+    var total: Double = 0.0
+
+    fun printProduct() {
+        println("Name: $name")
+        println("Price: $price")
+        println("Quantity Purchased: $quantityPurchased")
+        println("Total: $total")
+    }
+
```

Comment:
fun printProduct(){
    println("Name: $name Price: $price Quantity Purchased: $quantityPurchased Total: $total")
}

Reply:
One println() for all the properties. Okayy noted!